python/sql/core/Stored Procedures/dbo.sp_GBL_AIRS_Export_3_0.sql

```diff
@@ -1609,7 +1617,14 @@ SELECT
 				
 FROM GBL_BaseTable bt
 	INNER JOIN GBL_BaseTable_Description btd
-		ON bt.NUM=btd.NUM AND btd.LangID=@LangID
+		ON bt.NUM=btd.NUM
+			AND (
+				btd.LangID=@LangID 
+				OR (
+					@AutoIncludeSiteAgency=1
+					AND btd.LangID=(SELECT TOP 1 LangID FROM GBL_BaseTable_Description WHERE NUM=btd.NUM ORDER BY CASE WHEN LangID=@LangID THEN 0 ELSE 1 END, LangID))
+					AND EXISTS(SELECT * FROM dbo.GBL_BaseTable btl INNER JOIN dbo.GBL_BaseTable_Description btdl ON btdl.NUM = btl.NUM AND btdl.LangID=@LangID WHERE btl.ORG_NUM=bt.NUM)
+				)
```

Comment:
are the brackets here right? This doesn't have the same scoping as the Location case above